src/view/cards/wdr/wdr_content_reco_card.py

```diff
@@ -11,8 +13,54 @@ class WDRContentRecoCard(WDRContentCard):
     CARD_HEIGHT = 600
     IMAGE_HEIGHT = 200
 
-    def __init__(self, config):
-        super().__init__(config)
+    def _create_click_handler(self, external_id):
+        """
+        Create a click handler function that sets the value of a target widget
+        to the external_id. Uses view utils function to get widget by accessor name.
+
+        :param external_id: The external ID.
+        :return: The click handler function.
+        """
+        def click_handler(event):
+            target_widget = get_widget_by_accessor_function(self.reco_explorer_app_instance.config_based_nav_controls, "get_item_by_crid")
```

Comment:
Das is ne elegante Lösung, dass wir nicht das genaue Element kennen müssen.
Aber das findet ja nur das erste widget das im depth first search auftaucht.

Was passiert wenn es hypothetisch zwei widgets gibt die beide get_item_by_crid benutzen und wie beide befüllen müssen?

Reply:
Umbenennen ist erst mal ne gute Lösung.

Wenn mans ganz sauber machen will muss man wahrscheinlich die ganze Architektur von der Applikation überdenken. Weil aktuell ist der Filter-State 1:1 an die UI gekoppelt, das heißt es kann ohnehin keine zwei Widgets geben die den selben Filter manipulieren. 
Ist wahrscheinlich eine YAGNI Situation sein.